caseworker/advice/services.py

```
@@ -98,14 +116,20 @@ def get_advice_to_countersign(advice, caseworker):
     return grouped_user_advice
 
 
-def get_advice_to_consolidate(advice):
+def get_advice_to_consolidate(advice, caseworker):
     """For MOD consolidate, we need to be able to review advice from other
     teams - which is the only difference between this function and
     `get_advice_to_countersign`.
     """
     user_advice = filter_advice_by_level(advice, ["user"])
-    grouped_user_advice = group_advice_by_user(user_advice)
-    return grouped_user_advice
+    if caseworker["team"]["id"] == LICENSING_UNIT_TEAM:
+        advice_from_teams = filter_advice_by_teams(user_advice, LU_CONSOLIDATE_TEAMS)
+    elif caseworker["team"]["id"] == MOD_ECJU_TEAM:
+        advice_from_teams = filter_advice_by_teams(user_advice, MOD_CONSOLIDATE_TEAMS)
+    else:
+        advice_from_teams = user_advice
```

should we raise an exception here as only the above teams perform this action?

---

I think raising an exception would make sense, since displaying all the unfiltered advice could lead to some other problem?

---

ok